index.html

```diff
@@ -1,72 +1,264 @@
+            </p>
+            <ul class="tags-list-style-none">
+              <li class="tag-description1">HTML</li>
+              <li class="tag-description2">CSS</li>
+              <li class="tag-description3">JAVASCRIPT</li>
             </ul>
+            <button type="button" class="see-btn">See project</button>
         </div>
-            
-            
-
-            
-                
-                
-            
-            
-         
-           
-        
-    </main>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="card-container">
+        <img class="card-img" src="./vectors/Snapshoot Portfolio3.png" alt="image of my project" />
+        <div class="text-container">
+          <h2 class="card-title">Tonic</h3>
+            <div class="frame-2">
+              <p class="client">CANOPY</p>
+              <img src="./vectors/Counter.png" alt="Counter">
+              <p class="Role">Back End Dev</p>
+              <img src="./vectors/Counter.png" alt="Counter">
+              <p class="Role">2023</p>
+            </div>
+            <p class="card-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="tags-list-style-none">
+              <li class="tag-description1">HTML</li>
+              <li class="tag-description2">CSS</li>
+              <li class="tag-description3">JAVASCRIPT</li>
+            </ul>
+            <button type="button" class="see-btn">See project</button>
+        </div>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="card-container">
+        <img class="card-img" src="./vectors/Snapshoot Portfolio4.png" alt="image of my project" />
+        <div class="text-container">
+          <h2 class="card-title">Multi-Post <br> Stories</h3>
+            <div class="frame-2">
+              <p class="client">CANOPY</p>
+              <img src="./vectors/Counter.png" alt="Counter">
+              <p class="Role">Back End Dev</p>
+              <img src="./vectors/Counter.png" alt="Counter">
+              <p class="Role">2023</p>
+            </div>
+            <p class="card-description">
+              A daily selection of privately personalized reads; no accounts or sign-ups
+              required.
+            </p>
+            <ul class="tags-list-style-none">
+              <li class="tag-description1">HTML</li>
+              <li class="tag-description2">CSS</li>
+              <li class="tag-description3">JAVASCRIPT</li>
+            </ul>
+            <button type="button" class="see-btn">See project</button>
+        </div>
+      </div>
+  </section>
+  <div class="headline">
+    <h2 id="principal-title" class="color-blue fonts">About Myself
+    </h2>
+    <p class="p-headline fonts">I’m a software developer! I can help you build a product , feature or website Look
```

Comment:
- [ ] As required by the previous reviewer, please implement this rounded edge (top right) arrow_down_small in your "about-me" section, because it's part of the design and is currently missing in your project.
![227660477-a4d590f4-35e8-43d9-97b9-14995e3deb4d](https://user-images.githubusercontent.com/91142432/227690391-695755ba-7fad-4afc-a3c9-0f5f94828ad1.png)